### DISCUSSION TITLE
bootstrap 5 with display grid ?

### DISCUSSION THREAD

**Discussion by hassanbinsafdar** [twbs/bootstrap#31286] [General]
I want to ask is there you guys planned about including official display grid or not ?

**Comment by mdo** [comment]
We haven't solidified plans just yet. For now we've focused on bolstering our existing grid system, but we're continuing to experiment and try out new ideas.

**Reply by hassanbinsafdar** [reply]
ok great but i think if you guys added it would be great because browser support is awesome now . you can keep existing flexbox  grid but can give options of grid .
